Lib/test/test_cmd_line.py

```diff
@@ -37,6 +37,20 @@ def verify_valid_flag(self, cmd_line):
         self.assertNotIn(b'Traceback', out)
         self.assertNotIn(b'Traceback', err)
 
+    def test_help(self):
+        self.verify_valid_flag('-h')
+        self.verify_valid_flag('-?')
+        self.verify_valid_flag('--help')
+
+    def test_help_env(self):
+        self.verify_valid_flag('--help-env')
+
+    def test_help_xoptions(self):
+        self.verify_valid_flag('--help-xoptions')
+
+    def test_help_all(self):
+        self.verify_valid_flag('--help-all')
```

Comment:
The three new options could get a better test, say checking the first line for a well-known string (like help_usage on line 61 below)

Reply:
Let it not slow down you. If you add additional tests -- good (I would either add an optional parameter in `verify_valid_flag` for substrings which should occur or should not occur in the output, or make it returning the output string). But we can live without this and can add more tests later, for these and for other options.

Reply:
Adding parameters to `verify_valid_flag` doesn’t work (too many options to support checks for contains / matches / does not contain / in stdout / in stderr), but returning output is nice.